archives-updated.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Emits 'true' if a Fiji app bundle has been created since the most recent
```

Comment:
I find the name `archives-updated.sh` confusing. What do you think about renaming this to something like `fiji-archive-status.sh` and making the return strings more explicit like `up-to-date` vs `update-needed`?